visualizations/frontend/other/TTable.vue

```diff
@@ -313,7 +315,7 @@ export default {
     cellClasses: {
       type: String,
       default:
-        "border-b border-[#D3D3D9] align-top pt-[12px] pb-[17px] snykCell table-data",
+        "border-b border-[#D3D3D9] align-top py-[12px] break-all",
```

Comment:
The effect of using `break-all` is slightly different to `word-break: break-word`. In addition `break-words` (tailwind class), does not break the words as we would like. `break-all` seems to come the closest among the Tailwind utilities.  However, [MDN documentation on word-break ](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break) suggests we want the following to get the exact behaviour of `word-break: break-all;` 


```
word-break: normal;
overflow-wrap: anywhere;
```

Tailwind doesn't seem to have anything that matches this exactly. So maybe we create our own utility by renaming `snykCell` to something else utility like, or we just go with break-all and accept the differences for now.

Reply:
The snykCell with "word-break: break-word" was added to address an issue with very long project's forcing the column with the project to take up 60+% of the width because word-break: normal had no place to break the name up, so break-all should work just fine.

However now that I look at this again, the default styling was added here and made overridable because we weren't sure that it made sense to force the snyk table styling on every table instance, but now that we have been here a while I think it does make sense to have that styling be universal. Except for the break-word/all css which should only be applied to the issues detail table.

So I propose

1. removing snykCell and adding cellClass="break-all" to the issues detail table in both the page and pdf template
2. move "border-b border-[#D3D3D9] align-top py-[12px]" from the prop to class="border-b border-[#D3D3D9] align-top py-[12px]" on ~line 183 below or above where "generateCellClasses" is called.

Thoughts? If we agree I can make those changes real quick.

Reply:
I left `cellClasses` as is, to leave the props interface to the outside intact. I made the other changes. Though

Reply:
👍